angularjs-portal-home/src/main/webapp/my-app/layout/list/partials/home-list-view.html

```diff
@@ -1,4 +1,5 @@
 <div class="row portlet-frame">
+  <notification-bell mode="priority" class="visible-xs"></notification-bell>
```

Comment:
Is there any way we can make this not be in the header of every single page? Maybe in the app-header or in the frame page?